app/display/linearmeter/src/main/java/org/csstudio/display/extra/widgets/linearmeter/RTLinearMeter.java

```diff
@@ -1114,7 +1109,7 @@ else if (value >= high && value < hiHi) {
                                 gc.setPaint(needleColor);
                             }
                             // Draw the bar:
-                            gc.fillRect(marginLeft, currentIndicatorPosition, meterBreadth, linearMeterScale.getBounds().height-currentIndicatorPosition-marginBelow);
+                            gc.fillRect(marginLeft, currentIndicatorPosition, meterBreadth - 1, linearMeterScale.getBounds().height-currentIndicatorPosition-marginBelow);
```

Comment:
Found in several commits:

Math symbols (minus) could be surrounded by space for better readability.